docs/source/tutorials/indepth_overview.ipynb

```diff
@@ -40,7 +40,7 @@
     "You can use pip to install all packages required for this tutorial as follows:\n",
     "\n",
     "```\n",
-    "!pip install sklearn matplotlib\n",
+    "!pip install sklearn matplotlib datasets\n",
```

Comment:
```suggestion
    "!pip install sklearn matplotlib \n",
```

Don't need to explicitly state `datasets` as it will be installed with `cleanlab[datalab]`